.github/workflows/main.yml

```diff
@@ -0,0 +1,63 @@
+name: Tests
+
+on: [push]
+
+jobs:
+  run:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python-version: [2.7, 3.6, 3.7]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Add conda to $PATH
+      run: echo ::add-path::$CONDA/condabin
+
+    - name: Update conda on Mac
+      if: matrix.os == 'macos-latest'
+      run: |
+        # sudo required?
+        sudo conda update -y -n base conda setuptools
```

Comment:
Permissions might be a bit strict with actions. This is what we get without `sudo`:

```
NotWritableError: The current user does not have write permissions to a required path.
  path: /Users/runner/.conda/pkgs/urls.txt
  uid: 501
  gid: 20

If you feel that permissions on this path are set incorrectly, you can manually
change them by executing

  $ sudo chown 501:20 /Users/runner/.conda/pkgs/urls.txt

In general, it's not advisable to use 'sudo conda'.
```